nature/admin.py

```diff
@@ -43,7 +43,7 @@ class ObservationSeriesAdmin(admin.ModelAdmin):
 
 class ObservationInline(admin.TabularInline):
     model = Observation
-    fields = ('species', 'series', 'date', 'observer', 'number', 'description', 'notes',
+    fields = ('species', 'series', 'date', 'observer', 'quantity', 'description', 'notes',
```

Comment:
I don't see any reason to have this in a separate commit as it is clearly linked to the change of the model field and should therefore be in the same commit. It is a "fix" to a problem introduced earlier in the PR which means that it should be fixed in the initial commit :)

Reply:
true that :+1: